src/overlays/actors/ovl_Bg_Haka_Sgami/z_bg_haka_sgami.c

```diff
@@ -10,17 +10,25 @@
 
 #define THIS ((BgHakaSgami*)thisx)
 
+typedef enum {
+    /* 0 */ SCYTHE_TRAP_SHADOW_TEMPLE,
+    /* 1 */ SCYTHE_TRAP_SHADOW_TEMPLE_INVISIBLE,
+    /* 2 */ SCYTHE_TRAP_ICE_CAVERN
+} SpinningScytheTrapMode;
+
+#define SCYTHE_SPIN_TIME 0x20
```

Comment:
Since this is a frame count, decimal is probably more appropriate.
```suggestion
#define SCYTHE_SPIN_TIME 32
```